projects/core/src/occ/adapters/site-context/occ-site.adapter.ts

```diff
@@ -67,22 +68,14 @@ export class OccSiteAdapter implements SiteAdapter {
       );
   }
 
-  loadBaseSite(): Observable<BaseSite> {
-    const baseUrl = this.occEndpointsService.getBaseEndpoint();
-    const urlSplits = baseUrl.split('/');
-    const activeSite = urlSplits.pop();
-    const url = urlSplits.join('/') + '/basesites';
-
-    const params = new HttpParams({
-      fromString: 'fields=FULL',
-    });
-
+  loadBaseSites(): Observable<BaseSite[]> {
```

Comment:
I guess we should deprecrate the previous `loadBaseSite` method and not just throw it away

Reply:
I also wonder why we're dropping the `loadBaseSite` all together. Aren't we able to load individual baseSites anymore? Sounds good functionality to me, and also something we still use.

Reply:
There is no API to load one specific baseSite. We have to load all baseSites, and get one site from the list returned. Since, we have this `loadBaseSites` function, so I move this "get one site from the list" to facade service.
In `base-site.service`, there is `getAll()` which returns all baseSites, and also there is `getBaseSiteData()` which only return the active base site data.

Reply:
but I agree with you :). `load one base site` may require more fields than `load all baseSites`. So, I will add it back to api/connector/adapter.